index.html

```diff
@@ -54,6 +54,230 @@ <h1>
+                <h2>About <br> Myself</h2>
+            </div>
+            <div class="about-content">
+                <p class="about-content-p">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div>
+                <h4 class="connect">LET'S CONNECT</h4>
+                </div>
+                <ul class="socialmedia">
+                    <li>
+                        <a href="twitter.com"><img src="img/twitter.png" alt="twitter icon"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/messager.png" alt="messages icon"></a>
+                    </li>
+                    <li>
+                        <a href="linkedin.com"><img src="img/Linkedin icon.png" alt="Linkedin icon"></a>
+                    </li>
+                    <li>
+                        <a href="github.com"><img src="img/cat.png" alt="github icon"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/peace.png" alt="AngelList icon"></a>
+                    </li>
+                </ul>
+                <button type="button" class="my-button">Get My Resume</button>
+            </div>
+        </div>
+        <!-- About me part of the website.-->
+        <div class="aboutme">
+            <div class="skill-set">
+                <div class="languages">
+                    <p>Languages</p>
+                    <img src="./img/chevron-down.svg" alt="click to open a drop menu">
+                </div>
+                <div class="languages-list">
+                    <ul>
+                        <li>
+                            <img src="./img/Ellipse 1 (1).png" alt="JavaScript">
+                            <p>
+                                JavaScript
+                            </p>
+                        </li>
+                        <li>
+                            <img src="./img/Ellipse 1 (2).png" alt="HTML">
+                            <p>HTML</p>
+                        </li>
+                        <li>
+                            <img src="./img/Ellipse 1 (3).png" alt="CSS">
+                            <p>CSS</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="framework">
+                    <p class="flex">Frameworks</p>
+                    <img class=" flex arrow-img" src="./img/chevron-down.svg" alt=" Click to see the side bar">
+                    <hr>
+                </div>
+
+                <div class="skills">
+                    <p>Skills</p>
+                    <img class="arrow-img" src="./img/chevron-down.svg" alt="Click to see the side bar">
+                    <hr>
+                </div>
+
+            </div>
```

Comment:
- [ ] kindly consider implementing the addressed issues here as requested by the previous reviewer for better semantics. 😉